DatadogWebViewTracking/Sources/WebViewTracking.swift

```diff
@@ -25,28 +25,61 @@ import WebKit
 /// - Support users that have difficulty loading web pages on mobile devices
 public enum WebViewTracking {
 #if !os(tvOS)
-    /// Enables SDK to correlate Datadog RUM events and Logs from the WebView with native RUM session.
+    /// The Session Replay Configuration to capture records coming from the web-view.
     ///
+    /// Setting the Session Replay configuration in `WebViewTracking` will enable transmitting records from
+    /// the Datadog Browser SDK installed in the web page. Datadog will then be able to combine the native
```

Comment:
Likely the reader will not understand "record" as it's DD lingo:
```suggestion
    /// Setting the Session Replay configuration in `WebViewTracking` will enable transmitting replay data from
    /// the Datadog Browser SDK installed in the web page. Datadog will then be able to combine the native
```